reco_utils/recommender/rlrmc/RLRMCalgorithm.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
For people who has done great contributions like yours, we would love you to add your name in AUTHORS.md https://github.com/Microsoft/Recommenders/wiki/How-to-add-your-name-as-a-contributor-to-the-repo#add-your-name-in-authorsmd

Reply:
Thanks Miguel. Will do the same in some time.